boxsdk/client.py

```diff
@@ -124,17 +124,21 @@ def search(self, query, limit, offset, ancestor_folders=None, file_extensions=No
         :param ancestor_folders:
             Folder ids to limit the search to.
         :type ancestor_folders:
-            `list` of `unicode`
+            `list` of :class:`Folder`
```

Comment:
Good catch!

And actually we could be more flexible and call it `Iterable` of folder.